### PR TITLE
Adds file, prettyPrint and blackBox icons to dropdown

### PR DESCRIPTION
**Pull request by lukaszsobek**
Associated Issue: #4807

### Summary of Changes

* Adds file, prettyPrint and blackBox icons to dropdown

### Test Plan
dropdown-icons

### CODE REVIEW

File: src/components/Editor/Tabs.js [hunk 1]
```diff
@@ -317,13 +317,24 @@ class SourceTabs extends PureComponent<Props, State> {
...
   renderDropdownSource(source: SourceRecord) {
+    const sourceObj = source.toJS();
+    const sourceId = source.get("id");
+    let fileIconClass = "file";
+
+    if (isPretty(sourceObj)) {
+      fileIconClass = "prettyPrint";
+    } else if (source.get("isBlackBoxed")) {
+      fileIconClass = "blackBox";
+    }
```
Comment (jasonLaster) [inline src/components/Editor/Tabs.js]: lets move this to its' own function

Comment (lukaszsobek) [reply src/components/Editor/Tabs.js]: Done.

**Comment by codehag** [comment]
looks like a rebase might have gone wrong here!

File: src/components/shared/Dropdown.css [hunk 2]
```diff
@@ -48,6 +48,30 @@ html[dir="rtl"] .dropdown {
   background-color: var(--search-overlays-semitransparent);
 }
 
+.dropdown li img.prettyPrint {
+  -webkit-mask: url(/images/prettyPrint.svg) no-repeat;
```
Comment (codehag) [inline src/components/shared/Dropdown.css]: we have autoprefixer, so you should be able to use just mask :)

File: src/components/shared/Dropdown.css [hunk 3]
```diff
@@ -48,6 +48,30 @@ html[dir="rtl"] .dropdown {
...
+  mask: url(/images/prettyPrint.svg) no-repeat;
+  -webkit-mask-size: 100%;
+  mask-size: 100%;
+  background: var(--theme-highlight-blue);
+  width: 13px;
```
Comment (codehag) [inline src/components/shared/Dropdown.css]: this looks like a magic number, maybe we can make it a variable and name it?

File: src/components/shared/Dropdown.css [hunk 4]
```diff
@@ -48,6 +48,30 @@ html[dir="rtl"] .dropdown {
...
+  height: 13px;
+  margin-right: 5px;
+  vertical-align: middle;
+}
+
+.dropdown li img.blackBox {
```
Comment (codehag) [inline src/components/shared/Dropdown.css]: the styles here are largely the same, i think we can factor the similarity out into its own class, `.dropdown--icon` with two modifications `.dropdown--icon_blackbox` and `.dropdown--icon_prettyprint`

File: src/components/shared/Dropdown.css (hunk 2 above)
Comment (lukaszsobek) [reply src/components/shared/Dropdown.css]: Changed.

File: src/components/shared/Dropdown.css (hunk 3 above)
Comment (lukaszsobek) [reply src/components/shared/Dropdown.css]: Exported to variables.css as `--icon-size`

File: src/components/shared/Dropdown.css (hunk 4 above)
Comment (lukaszsobek) [reply src/components/shared/Dropdown.css]: using combined `.dropdown-icon` that holds the majority of the css + `.blackBox`, `.prettyPrint` & `.file`. It seems that you cannot move the mask size declaration to a different class than declaring the actual mask, interesting.

**Pull request merged**